### PR TITLE
Refactored some functions in printer.py

### PR DESCRIPTION
**Pull request by KulkarniSuraj**
Just made minor changes in a file. I hope this  code doesn't break anything
test it before merging

### CODE REVIEW

File: covid19/printer.py [hunk 1]
```diff
@@ -34,18 +34,10 @@ def get_in_info(self, message):
...
-
-def get_tab(count=1):
-    i = 0
-    while i < count:
-        print("\t", end="")
-        i += 1
+new_lines = lambda count=1: print(end="\n"*count)
+
+
+get_tab = lambda count=1: print("\t"*count, end="")
```
Comment (asprazz) [inline covid19/printer.py:40]: Can we directly call this variables from outside like functions ?

Comment (KulkarniSuraj) [reply covid19/printer.py:40]: Yes, we can. They are lambda functions, just like arrow functions in javascript.
you can test them in python shell

**Comment by asprazz** [comment]
@KulkarniSuraj Next time please use a seperate branch for pull request or your every change will get pushed here. :smile:

**Comment by KulkarniSuraj** [comment]
> @KulkarniSuraj Next time please use a seperate branch for pull request or your every change will get pushed here. 😄

😂 I still feel new to github. The solution I offered for formatting might not be the best but it will work on all platforms because it is just formatting of value

File: covid19/printer.py [hunk 2]
```diff
@@ -35,17 +35,11 @@ def get_in_info(self, message):
 
 
 def new_lines(count=1):
-    i = 0
-    while i < count:
-        print("")
-        i += 1
+    print(end="\n"*count)
```
Comment (asprazz) [inline covid19/printer.py]: I changed this in recent verion @Naaru-01 I don't know why it was failing :laughing:

**Pull request closed without merging**